autoswitch_virtualenv.plugin.zsh

```diff
@@ -320,13 +320,14 @@ function enable_autoswitch_virtualenv() {
     autoload -Uz add-zsh-hook
     disable_autoswitch_virtualenv
     add-zsh-hook chpwd check_venv
+    add-zsh-hook precmd check_venv
```

Comment:
problem with this is that it's going to be running the same thing twice. But it indicates to me what the problem likely is